configs/server/forseti_conf_server.yaml.in

```diff
@@ -19,17 +19,36 @@ inventory:
     domain_super_admin_email: {DOMAIN_SUPER_ADMIN_EMAIL}
 
     api_quota:
-        # The pre-populated values are the defaults from GCP.
-        max_admin_api_calls_per_100_seconds: 1500
-        max_appengine_api_calls_per_second: 20
-        max_bigquery_api_calls_per_100_seconds: 17000
-        max_cloudbilling_api_calls_per_60_seconds: 300
-        max_compute_api_calls_per_second: 20
-        max_container_api_calls_per_100_seconds: 1000
-        max_crm_api_calls_per_100_seconds: 400
-        max_iam_api_calls_per_second: 20
-        max_servicemanagement_api_calls_per_100_seconds: 200
-        max_sqladmin_api_calls_per_100_seconds: 100
+        admin:
+          max_calls: 14
+          period: 1.0
+        appengine:
+          max_calls: 18
+          period: 1.0
+        bigquery:
+          max_calls: 160
+          period: 1.0
+        cloudbilling:
+          max_calls: 5
+          period: 1.2  # Include some room for retries
+        compute:
+          max_calls: 18
+          period: 1
```

Comment:
I recommend having period always be set as a float in our standard configuration (1.0). Of course it works the same on the backend, but it makes it more obvious to others who want to manage the configuration.